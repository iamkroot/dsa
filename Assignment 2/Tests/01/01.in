5 6
0 1 1
1 2 2
2 3 1
3 0 1
4 0 1
3 4 2
