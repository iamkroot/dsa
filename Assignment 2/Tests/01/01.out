Cycles with minimum weight 5 are:
0->1->2->3->0
